ReactWindows/ReactNative.Tests/Bridge/ReactInstanceTests.cs

```diff
@@ -81,14 +81,14 @@ public class ReactInstanceTests
 
             var instance = await DispatcherHelpers.CallOnDispatcherAsync(() => builder.Build());
             reactContext.InitializeWithInstance(instance);
-            await DispatcherHelpers.RunOnDispatcherAsync(() => instance.Initialize());
+            await DispatcherHelpers.RunOnDispatcherAsync(async () => await instance.InitializeAsync());
```

Comment:
I think we might need to change this to CallOnDispatcherAsync so it waits for the completion of the inner task.

Reply:
Done